src/multi-selector.js

```diff
@@ -103,16 +103,23 @@ export default class MultiSelector extends React.Component {
       lastModifiedItem: null,
       invalid: false,
     };
+
+    document.addEventListener('click', this.close);
```

Comment:
NIT - Why was this moved into the constructor? Semantically  it might make more sense inside `componentDidMount`

Reply:
I can move it to `componentDidMount`